js/fetch_test.ts

```diff
@@ -47,6 +47,26 @@ testPerm({ net: true }, async function responseClone() {
   }
 });
 
+// TODO(ry) The following tests work but are flaky. There's a race condition
```

Comment:
Unintended commit from something else?

Reply:
Intended. These tests are flaky and causing CI to fail.